flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/declarative/DeclarativeSchedulerNG.java

```diff
@@ -170,7 +168,7 @@
     private State state = new Created();
 
     public DeclarativeSchedulerNG(
-            JobGraph jobGraph,
+            JobGraph jobInformation,
```

Comment:
search&replace was a bit too eager ;)

Reply:
ah, that was the argument the refactoring tool asked me about. Should have looked closer 🤦